frontend/app/src/components/history/events/HistoryEventsList.vue

```diff
@@ -26,115 +26,98 @@ const emit = defineEmits<{
 }>();
 
 const PER_BATCH = 6;
-const currentLimit = ref<number>(0);
-
+const currentLimit = ref<number>(PER_BATCH);
 const { t } = useI18n();
-
 const { eventGroup, allEvents } = toRefs(props);
 
 const events = computed<HistoryEventEntry[]>(() => {
   const all = get(allEvents);
   const eventHeader = get(eventGroup);
   if (all.length === 0)
     return [eventHeader];
-
   const eventIdentifierHeader = eventHeader.eventIdentifier;
-  const filtered = all
+  return all
     .filter(({ eventIdentifier, hidden }) => eventIdentifier === eventIdentifierHeader && !hidden)
     .sort((a, b) => Number(a.sequenceIndex) - Number(b.sequenceIndex));
-
-  if (filtered.length > 0)
-    return filtered;
-
-  return [eventHeader];
 });
 
-const ignoredInAccounting = useRefMap(eventGroup, ({ ignoredInAccounting }) => !!ignoredInAccounting);
+const ignoredInAccounting = computed(() => !!get(eventGroup).ignoredInAccounting);
 
 const showDropdown = computed(() => {
-  const length = get(events).length;
-  return (get(ignoredInAccounting) || length > PER_BATCH) && length > 0;
+  const length = events.value.length;
+  return (ignoredInAccounting.value || length > PER_BATCH) && length > 0;
```

Comment:
@cleanerzkp Please make sure don't use `.value`, keep `get` and `set`.
every usage in this file